[2024-01-31 12:48:31,462] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-01-31T12:48:26.836351+00:00 [queued]>
[2024-01-31 12:48:31,468] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-01-31T12:48:26.836351+00:00 [queued]>
[2024-01-31 12:48:31,468] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 12:48:31,468] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-01-31 12:48:31,468] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 12:48:31,477] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): import_file_to_mysql> on 2024-01-31 12:48:26.836351+00:00
[2024-01-31 12:48:31,482] {standard_task_runner.py:52} INFO - Started process 4917 to run task
[2024-01-31 12:48:31,484] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'import_file_to_mysql', 'manual__2024-01-31T12:48:26.836351+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpst2hq15y', '--error-file', '/tmp/tmp8kl1r_wi']
[2024-01-31 12:48:31,485] {standard_task_runner.py:80} INFO - Job 103: Subtask import_file_to_mysql
[2024-01-31 12:48:31,520] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-01-31T12:48:26.836351+00:00 [running]> on host d99f9e65d396
[2024-01-31 12:48:31,565] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=import_file_to_mysql
AIRFLOW_CTX_EXECUTION_DATE=2024-01-31T12:48:26.836351+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-31T12:48:26.836351+00:00
[2024-01-31 12:48:32,299] {logging_mixin.py:115} INFO - Error install json
[2024-01-31 12:48:35,257] {logging_mixin.py:115} INFO - Successfully install pandas
[2024-01-31 12:48:38,039] {logging_mixin.py:115} INFO - Successfully install pymysql
[2024-01-31 12:48:38,414] {python.py:173} INFO - Done. Returned value was: import to mysql
[2024-01-31 12:48:38,430] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_postgresql, task_id=import_file_to_mysql, execution_date=20240131T124826, start_date=20240131T124831, end_date=20240131T124838
[2024-01-31 12:48:38,449] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-31 12:48:38,470] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
